pom.xml

```diff
@@ -1034,6 +1034,67 @@
                     <configuration>
                         <buildOutputDirectory>eclipse-build</buildOutputDirectory>
                         <downloadSources>true</downloadSources>
+                        <additionalConfig>
+							<file>
```

Comment:
Looks like you are using tabs from this line onwards, can you switch to spaces instead?